webpack.config.js

```diff
@@ -30,3 +24,33 @@ module.exports = {
     },
   },
 };
+
+const nodeConfig = {
+  ...baseConfig,
+  target: "node",
+  output: {
+    library: {
+      type: "umd",
+    },
+    filename: "index.node.js",
```

Comment:
```suggestion
    filename: "index.cjs",
```
cjs is a better descriptor. Implying that it's for node is misleading since node can cope just fine with ES modules.